pootle/apps/pootle_statistics/migrations/0011_cleanup_submissions_squashed_0023_remove_scorelog.py

```diff
@@ -0,0 +1,191 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.7 on 2017-05-11 04:59
+from __future__ import unicode_literals
+
+from django.conf import settings
+from django.db import migrations
+
+from pootle.core.batch import Batch
+
+
+# Old types
+REVERT_TYPE = 2
+SUGG_ACCEPT_TYPE = 3
+SUGG_ADD_TYPE = 8
+SUGG_REJECT_TYPE = 9
+UNIT_CREATE_TYPE = 10
+
+# Fields
+SOURCE_FIELD = 1
+TARGET_FIELD = 2
+STATE_FIELD = 3
+
+# Remaining types
+MUTE_TYPE = 6
+UNMUTE_TYPE = 7
+
+NEW_MUTED = 0
+NEW_UNMUTED = 1
+NEW_CHECK_FIELD = 5
+NEW_WEB_TYPE = 1
+
+# 2, 3, 8, 9, 10
+OLD_TYPES = [
+    REVERT_TYPE,
+    SUGG_ACCEPT_TYPE,
+    SUGG_ADD_TYPE,
+    SUGG_REJECT_TYPE,
+    UNIT_CREATE_TYPE]
+
+
+CLEANUP_STORE_DATA_SQL = (
+    "UPDATE `pootle_store_data` "
+    " INNER JOIN `pootle_app_submission` "
+    "  ON `pootle_app_submission`.`id` = `pootle_store_data`.`last_submission_id` "
+    " SET `last_submission_id` = NULL"
+    " WHERE `pootle_app_submission`.`type` in (%s)")
+
+CLEANUP_TP_DATA_SQL = (
+    "UPDATE `pootle_tp_data` "
+    " INNER JOIN `pootle_app_submission` "
+    "  ON `pootle_app_submission`.`id` = `pootle_tp_data`.`last_submission_id` "
+    " SET `last_submission_id` = NULL"
+    " WHERE `pootle_app_submission`.`type` in (%s)")
+
+CLEANUP_SQL = (
+    "DELETE FROM `pootle_app_submission` "
+    " WHERE (`unit_id` IS NULL"
+    "        OR `pootle_app_submission`.`type` in (%s))")
+
+
+def cleanup_subs_with_sql(schema_editor):
+    cursor = schema_editor.connection.cursor()
+    _old_types = ", ".join(str(t) for t in OLD_TYPES)
+    cursor.execute(CLEANUP_STORE_DATA_SQL % _old_types)
+    cursor.execute(CLEANUP_TP_DATA_SQL % _old_types)
```

Comment:
Why these two SQL queries above don't have a matching ORM call in `cleanup_subs_with_orm`?